Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners_test.py

```diff
@@ -199,55 +176,135 @@ def test_aggregate(owners, expected_out):
     assert sorted(aggregate(owners), key=lambda x: sorted(x.items())) == sorted(expected_out, key=lambda x: sorted(x.items()))
 
 
-@pytest.mark.parametrize('deduplicated, expected_out', [
-    # equal counts
-    (
-        [
-            {'Name': 'aa', 'Email': 'email1@gmail.com', 'Source': 'source1 | source2', 'Timestamp': '2', 'Count': 2},
-            {'Name': 'aa', 'Email': '', 'Source': 'source3 | source4', 'Timestamp': '4', 'Count': 2},
-        ],
-        [
-            {'Name': 'aa', 'Email': 'email1@gmail.com', 'Source': 'source1 | source2', 'Timestamp': '2', 'Ranking Score': 1.0},
-            {'Name': 'aa', 'Email': '', 'Source': 'source3 | source4', 'Timestamp': '4', 'Ranking Score': 1.0},
-        ]
-    ),
-    # unequal counts
-    (
-        [
-            {'Name': 'aa', 'Email': 'email1@gmail.com', 'Source': 'source1', 'Timestamp': '2', 'Count': 1},
-            {'Name': 'aa', 'Email': '', 'Source': 'source3 | source4', 'Timestamp': '4', 'Count': 2},
-        ],
-        [
-            {'Name': 'aa', 'Email': 'email1@gmail.com', 'Source': 'source1', 'Timestamp': '2', 'Ranking Score': 0.5},
-            {'Name': 'aa', 'Email': '', 'Source': 'source3 | source4', 'Timestamp': '4', 'Ranking Score': 1.0},
-        ]
-    ),
-    # empty owners
-    (
-        [],
-        []
-    )
+@pytest.mark.parametrize('scores,expected_out', [
+    ([0.5, 1], [0.67, 0.83]),
+    ([1, 10, 100], [.5, .55, .95]),
+    ([], []),
 ])
-def test_score(deduplicated, expected_out):
-    assert score(deduplicated) == expected_out
+def test_normalize_scores(scores, expected_out):
+    assert np.allclose(normalize_scores(scores), expected_out, atol=0.01)
+
+
+def test_score_base_case(mocker):
+    """
+    Test the default scoring case
+    """
+    model_mock = Mock()
+    model_mock.predict.return_value = np.array([4, 7, 31, 0.33, 15])
```

Comment:
where did these numbers come from? and the ones in line 215?

If the right value of what we are testing isn't readily apparent to the reader of the test, we should change something about the test -- tests should be verifying obvious functionality. some options might be:
1. break apart the test so the source of the numbers is apparent (maybe into multiple tests -- e.g., maybe we test featurization of the `owners` values separate from model prediction)
2. add enough context so the number source is apparent
3. make assertions about something that *is* apparent from the data (like who is the most / least likely to be the best owner point-of-contact, or that we do get a numeric answer)

you've already got 1, 2, and 3 covered below -- so after reviewing the rest of this doc, I'm wondering, does this test cover something we aren't testing elsewhere? could we simplify it?